core/controllers/collection_editor.py

```diff
@@ -172,12 +176,18 @@ def put(self, collection_id):
             raise self.InvalidInputException(e)
 
         # Retrieve the updated collection.
+        collection = (
+            collection_services.get_collection_by_id(collection_id))
+
         collection_dict = (
             summary_services.get_learner_collection_dict_by_id(
                 collection_id, self.user_id, allow_invalid_explorations=True))
 
-        # Send the updated collection back to the frontend.
-        self.values.update(collection_dict)
+        # Return updated collection.
+        self.values.update({
+            'collection': collection_dict
```

Comment:
I feel like the comment and update could be reverted back to what they were originally:

```python
# Send the updated collection back to the frontend.
self.values.update(collection_dict);
```

WDYT?